server/src/main/java/org/elasticsearch/action/admin/cluster/stats/ClusterStatsNodes.java

```diff
@@ -769,4 +774,64 @@ public XContentBuilder toXContent(final XContentBuilder builder, final Params pa
 
     }
 
+    static class IndexPressureStats implements ToXContentFragment {
+
+        private IndexingPressureStats indexingPressureStats = new IndexingPressureStats(
```

Comment:
Could we make this field `final` (and therefore assign it just once in the constructor) so that this stats object is properly immutable?